### ISSUE TITLE
[MacOS 13+] libintl.8.dylib missing when calling mrv2 via command-line 

### ISSUE THREAD

**Issue by p2or** [ggarra13/mrv2#55]
Hi @ggarra13,

Just noticed that when I call _mrv2_ from the command-line and try to pass an arbitrarily image sequence using `/Applications/mrv2.app/Contents/Resources/bin/mrv2 '/tmp/seq_####.exr'` or `/Applications/mrv2.app/Contents/Resources/bin/mrv2 '/tmp/seq_%04d.exr'`, I get the following error message:

```
dyld[3242]: Library not loaded: @rpath/libintl.8.dylib
  Referenced from: <1A7CEEF9-E0FE-3758-8831-6F330C9DF7EB> /Applications/mrv2.app/Contents/Resources/bin/mrv2
  Reason: tried: '/Users/gga/code/applications/mrv2/BUILD-Darwin-amd64/Release/install/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/gga/code/applications/mrv2/BUILD-Darwin-amd64/Release/install/lib/libintl.8.dylib' (no such file), '/usr/local/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/lib/libintl.8.dylib' (no such file), '/Users/gga/code/applications/mrv2/BUILD-Darwin-amd64/Release/install/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/gga/code/applications/mrv2/BUILD-Darwin-amd64/Release/install/lib/libintl.8.dylib' (no such file), '/usr/local/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/lib/libintl.8.dylib' (no such file), '/usr/local/lib/libintl.8.dylib' (no such file), '/usr/lib/libintl.8.dylib' (no such file, not in dyld cache)
```
Just wondering whether command-line calls are implemented yet, since calling `mrv2 -h` doesn't work either. Should this work out of the box (and the library is just missing) or am I doing something wrong?

Thanks a lot for _mrv2_,
Christian

**Comment by ggarra13** [comment]
Hi, @p2or.  For command-line usage, you need to call the startup shell script, that will set up all the paths for finding the libraries.

/Applications/mrv2.app/Contents/Resources/bin/mrv2.sh <your_file_here>

Hope that helps.

**Comment by p2or** [comment]
Hi @ggarra13,

thanks for the quick reply, I should have mentioned that I've tried that too. Sorry to say that the output is the same:

```
dyld[7402]: Library not loaded: @rpath/libintl.8.dylib
  Referenced from: <1A7CEEF9-E0FE-3758-8831-6F330C9DF7EB> /Applications/mrv2.app/Contents/Resources/bin/mrv2
  Reason: tried: '/Users/gga/code/applications/mrv2/BUILD-Darwin-amd64/Release/install/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/gga/code/applications/mrv2/BUILD-Darwin-amd64/Release/install/lib/libintl.8.dylib' (no such file), '/usr/local/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/lib/libintl.8.dylib' (no such file), '/Users/gga/code/applications/mrv2/BUILD-Darwin-amd64/Release/install/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/Users/gga/code/applications/mrv2/BUILD-Darwin-amd64/Release/install/lib/libintl.8.dylib' (no such file), '/usr/local/lib/libintl.8.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/lib/libintl.8.dylib' (no such file), '/usr/local/lib/libintl.8.dylib' (no such file), '/usr/lib/libintl.8.dylib' (no such file, not in dyld cache)
/Applications/mrv2.app/Contents/Resources/bin/mrv2.sh: line 38:  7402 Abort trap: 6           sh -c "${dir}/bin/mrv2 $params"
```
Is there anything else I can try?

**Comment by ggarra13** [comment]
Ok.  I was able to reproduce your problem.  Find attached a new start up script.   Replace /Applications/mrv2.app/Contents/MacOS/mrv2  with this new script.   

[mrv2.zip](https://github.com/ggarra13/mrv2/files/11642693/mrv2.zip)

Make sure it is executable.  That is:

cp <new_script> /Applications/mrv2.app/Contents/MacOS/mrv2 
chmod a+x /Applications/mrv2.app/Contents/MacOS/mrv2

Then run it like:

/Applications/mrv2.app/Contents/MacOS/mrv2 test.0001.exr

**Comment by p2or** [comment]
Great, can confirm! Works like a charm, thank you very much for this quick fix @ggarra13. I would leave it up to you closing the issue when patching it, otherwise just let me know. 

Cheers, 
Christian

**Comment by ggarra13** [comment]
I added some modifications to the script so it works fine from any directory or from any symlink.  Find the modified script attached:

[mrv2.zip](https://github.com/ggarra13/mrv2/files/11643004/mrv2.zip)

**Comment by p2or** [comment]
Awesome. Tested, works just as good.

**Comment by ggarra13** [comment]
@p2or I forgot to ask you, since you are on macOS 13.  Can you use mrv2's file requester instead of the native file requester (Preferences->File Requester->Use Native File Chooser off) to access your private directories (ie. Documents, Downloads and Desktop)?  When you use it for the first time macOS should ask you for permission to read them, or at least, that's what I am hoping for.  If mrv2 cannot list them with mrv2's file requester instead of the native one, it would be helpful for me to know.

**Comment by p2or** [comment]
Hi @ggarra13,

yes, works exactly in that way you describe. MacOS asks for permissions, but everything is displayed as expected:

<img width="904" alt="Screenshot 2023-06-03 at 19 49 34" src="https://github.com/ggarra13/mrv2/assets/512368/4c92081f-15bb-4645-967f-9e4b404bae6d">

_Except_ that the property is **turned off** by default and I can't get the MacOS _File Chooser_ back once the I enabled it. Although I have _turned it off_, _mrv2_ doesn't seem to store the value, no matter what I do:

<img width="857" alt="Screenshot 2023-06-03 at 19 56 33" src="https://github.com/ggarra13/mrv2/assets/512368/d53dadfb-cff0-4c9f-81ea-142d6aadc51e">

Hope this helps. If you prefer having a separate issue for it, just let me know.

**Comment by ggarra13** [comment]
I think you might be confused.

Turning it OFF disables the macOS native file chooser.  Turning it ON enables the mrv2 built-in file chooser.

The mrv2 built-in file chooser allows you to drag it and resize it.  The macOS native one does not.

There should be thumbnails of the images with the mrv2 built-in file chooser, but I never tried them without a base name in their name.  That's likely why you are not getting thumbnails in the picture you sent me above.

**Comment by p2or** [comment]
Hi @ggarra13,

removed the entire `.filmaura` user folder to avoid any confusion, to see if I've tampered with the original settings and messed it up at some point before. You're absolutly right, by default 'Use Native File Chooser' is disabled and I'm getting _mrv2_ built-in one. When enabled I get the 'File Chooser' provided by the OS _(not quite sure why the native version was displayed yesterday, even though the property was disabled)_. Anyway, seems all good now, sorry for wasting your time on that - If this issue comes up again and I can reproduce it, I'll let you know.

_Don't worry, the thumbnails work. Maybe it's hard to see, but I disabled the 'Preview files' property for the screenshot._

Cheers,
Christian
